### PR TITLE
rpcsrv: allow any Origin in WS connections if EnableCORSWorkaround

### PR DESCRIPTION
**Pull request by roman-khimov**
Break origin checks even more. Alternative to #2772.

### CODE REVIEW

File: pkg/services/rpcsrv/server.go [hunk 1]
```diff
@@ -282,11 +279,16 @@ func New(chain Ledger, conf config.RPC, coreServer *network.Server,
 	if orc != nil {
 		oracleWrapped.Store(&orc)
 	}
+	var wsOriginChecker func(*http.Request) bool
+	if conf.EnableCORSWorkaround {
```
Comment (AnnaShaleva) [inline pkg/services/rpcsrv/server.go:283]: Do we need to update documentation for this option? There's only a couple of words about CORS, and this code is related to origin check.

**Review by AnnaShaleva — COMMENTED** [review]
Otherwise LGTM.

File: pkg/services/rpcsrv/server.go (hunk 1 above)
Comment (roman-khimov) [reply pkg/services/rpcsrv/server.go:283]: Updated!

**Review by AnnaShaleva — APPROVED** [review]
Pretty clear.

**Pull request merged**